ingestion/functions/retrieval/requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
no pinning to a particular version? It might break.

Reply:
Good point, not sure why I waffled on that.